lecture_04/assignment_03/hanbing_zhao/assignment-03.py

```diff
@@ -0,0 +1,65 @@
+"""Assignment 03: Using inverse kinematics
+"""
+import os
+import compas
+from compas_fab.backends import RosClient
+from compas_fab.robots import Configuration
+
+from compas.geometry import Frame
+from compas.geometry import Point
+from compas.geometry import Vector
+
+import math
+import json
+
+
+# Step 1: Inside this function, complete the main part of the solution for the assignment:
+#  - Taking a robot and a list of frames as parameter, calculate a feasible configuration for each of the frames
+#  - Try to find an optimal start_configuration for each so that the motion from one config to the next is minimized
+def calculate_ik_for_frames(robot, frames):
+    configs = []
+    for i in range(len(frames)):
+        config = Configuration.from_revolute_values([i * math.pi/2, 0., 0.])
+        config.joint_values
+        configs.append(config)
+    return configs
+
+
+# Step 2: store all found configurations in a JSON file using compas.json_dump or compas.json_dumps
+def store_configurations(configurations, filename):
+<<<<<<< HEAD
```

Comment:
make sure you clean up these things, these are artifacts of git merges and it will make the python interpreter fail